### PR TITLE
ConsentSJF profile

### LINKED ISSUE #28: Talk to HL7 Sweden about creating PDL supporting profiles
The need to create PDL supporting profiles is highly prioritized at Cambio for Q3-Q4 2022. This is something that would be best described on a national level, so we should suggest to HL7 Sweden that they should take lead on this.

The business concepts that need representation are:

- Blocks (inner and outer)
- Consents to shared electronic health records
- Access logs as defined by PDL

### PR DESCRIPTION
**Pull request by peturv**
Moving file to separate branch to fix build.
Fixed a couple of the errors

### CODE REVIEW

File: input/fsh/ConsentSJF.fsh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* provision.actor ^slicing.discriminator.path = "role.coding.code"
+* provision.actor ^slicing.description = "Slices for different actor roles"
+* provision.actor ^slicing.ordered = false
+* provision.actor ^slicing.rules = #closed
+* provision.actor contains
+    grantor 1..1 MS and
+    grantee 1..1 MS and
+    primaryauthor 0..1 MS
+
+* provision.actor[grantor].role.coding.system = ROLE_CODE_SYSTEM (exactly)
```
Comment (peturv) [inline input/fsh/ConsentSJF.fsh]: ROLE_CODE_SYSTEM is probably an alias that has not been included in the original commit
Either add ROLE_CODE_SYSTEM with correct URI to the aliases.fsh file or replace with correct value directly

File: input/fsh/ConsentSJF.fsh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+* provision.actor[grantor].role.coding.code 1..
+* provision.actor[grantor].role.coding.code = #GRANTOR (exactly)
+* provision.actor[grantor].reference only Reference(Patient or RelatedPerson)
+* provision.actor[grantor].reference.identifier 1..
+* provision.actor[grantor].reference.identifier.system 1..
+* provision.actor[grantor].reference.identifier.value 1..
+
+* provision.actor[grantee].role.coding.system = ROLE_CODE_SYSTEM (exactly)
```
Comment (peturv) [inline input/fsh/ConsentSJF.fsh]: ROLE_CODE_SYSTEM is probably an alias that has not been included in the original commit
Either add ROLE_CODE_SYSTEM with correct URI to the aliases.fsh file or replace with correct value directly

**Review by peturv — COMMENTED** [review]
2 Errors that I can find. Once they are fixed, we should be able to merge the profile in to master.
@vjernelov could you check the original files and see what is missing? (See comments below)

File: input/fsh/ConsentSJF.fsh (hunk 1 above)
Comment (vjernelov) [reply input/fsh/ConsentSJF.fsh]: Fixed that in the branch "28-talk-to-hl7-sweden..."

File: input/fsh/ConsentSJF.fsh (hunk 2 above)
Comment (vjernelov) [reply input/fsh/ConsentSJF.fsh]: Fixed that in the branch "28-talk-to-hl7-sweden..."

**Pull request merged**